cuchem/cuchem/benchmark/megamolbart.py

```diff
@@ -231,10 +228,8 @@ def main(cfg):
             if iter_val in model_dict:
                 estimator, param_dict = model_dict[iter_val]
 
-            result = metric.calculate(smiles_dataset=smiles_dataset,
-                                      fingerprint_dataset=fingerprint_dataset,
+            result = metric.calculate(fingerprint_dataset=fingerprint_dataset,
                                       top_k=iter_val, # TODO IS THIS A BUG
```

Comment:
@rilango I think this is a bug. Should make sure we fix after the refactor.

Reply:
Yes... I will be revisiting all arguments to calculate again once we move everything out into a separate module.